packages/usa-input-mask/src/content/usa-input-mask-date-short.json

```diff
@@ -0,0 +1,10 @@
+{
+  "hint": "For example, 01/01/1970",
+  "id": "us-date",
+  "inputmode": "numeric",
+  "label": "US Date",
```

Comment:
Should all labels with `US` instead say `U.S.`? It seems to be how we use it elsewhere on the site. Not sure if there is a universal stance on this.